go.mod

```diff
@@ -2,4 +2,17 @@ module github.com/elastic/go-structform
 
 go 1.12
 
-require github.com/stretchr/testify v1.7.0
+require (
+	github.com/Masterminds/goutils v1.1.1 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
+	github.com/elastic/go-ucfg v0.8.3 // indirect
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/huandu/xstrings v1.3.2 // indirect
+	github.com/imdario/mergo v0.3.12 // indirect
+	github.com/mitchellh/copystructure v1.2.0 // indirect
+	github.com/stretchr/testify v1.7.0
+	github.com/urso/mktmpl v0.0.0-20180112143305-7705185786e8 // indirect
+	golang.org/x/tools v0.1.5 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+)
```

Comment:
I presume most of these dependencies are introduced by the mktmpl tool.

For tools I would prefer to [bingo](https://github.com/bwplotka/bingo) approach instead of 'polluting' the go.mod file.

Reply:
Yes, that's why they're there. I'm not convinced by the claims made by bingo, but is there anything that you would like me to do with this?

Reply:
Let's add a `tools.go` to stabilize go.mod. Thanks

Reply:
Done.